### DISCUSSION TITLE
Clarifications about ONNX and openvino opset number

### DISCUSSION THREAD

**Discussion by lodm94** [openvinotoolkit/openvino#13966] [Q&A]
Hi all, 
i don't understand which is the relationship, if any, between ONNX opset version and openvino's one. I tried reading both documentation but i just got more confused.

I find very simple and straightforward to deploy a model in IR starting from ONNX. Still, when i move from PyTorch to ONNX, i need to set the opset version parameter. I can't find which is the suitable value if i need to convert that ONNX to IR and perform inference inside [this](https://hub.docker.com/r/openvino/ubuntu20_runtime) openvino runtime environment.

Any help would be very welcome
Best

**Comment by tomdol** [comment]
OV opsets are and entirely different thing than ONNX opsets. When you convert from PyTorch to ONNX you're basically compiling the same model between two formats. The same thing happens when you feed your ONNX model to OV. Currently the OV master branch is aligned with ONNX opset 17 (and below). This can be checked in this line https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/CMakeLists.txt#L18
We don't support all ONNX opset 17 operators though.
If you have any other questions please keep asking.

**Comment by YOODS-Xu** [comment]
@tomdol 
Could you give us a compability matrix document please?
